gadget/internal/mkfs.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
-type MkfsFunc func(imgFile, label, contentsRootDir string, deviceSize quantity.Size) error
+type MkfsFunc func(imgFile, label, contentsRootDir string, deviceSize, sectorSize quantity.Size) error
```

Comment:
one more parameter and we'll likely need to pack some of them into a struct, otherwise it's slowly getting out of hand

Reply:
I'll change it if we end up needing more, for now I think it is still somewhat readable